packages/test/test-end-to-end-tests/package.json

```diff
@@ -144,6 +144,20 @@
 	},
 	"fluidBuild": {
 		"tasks": {
+			"compile": {
+				"dependsOn": [
+					"...",
+					"^build:esnext"
+				],
+				"script": false
+			},
+			"ci:build": {
+				"dependsOn": [
+					"...",
+					"^build:esnext"
+				],
+				"script": false
+			},
```

Comment:
These changes might eventually be needed at the top level, but for now only this project needs it. Keeping the settings here lets fluid-build optimize task distribution for all the other packages.